website/docs/using-unleash/troubleshooting/https.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Configuring Unleash to run over HTTPS
+---
+
+Preferred methods for setting up HTTPS in self-hosted instances, from highly recommended to those requiring more caution:
+
+## Load Balancer
+
+The best choice is to use a load balancer. 
+A load balancer from a cloud provider renews the HTTPS certificates for you and keeps the data safe when it moves between the internet and your server.
+Also, your cloud provider's private network between your load balancer and the application is already encrypted.
+
+## Sidecar
+
+If you're using something like Kubernetes and need HTTPS to be handled right next to your Unleash app, use a sidecar pattern. 
+This method keeps the HTTPS handling separate from the Unleash application logic.
+Tools like Istio, HAProxy, or Nginx can help by automatically updating certificates.
```

Comment:
could we add links to these tools?

Reply:
yes

Reply:
[Envoy](https://www.envoyproxy.io/) is quite popular too these days.